proto/destination.proto

```diff
@@ -49,11 +49,11 @@ import "net.proto";
 service Destination {
   // Given a destination, return all addresses in that destination as a long-
   // running stream of updates.
-  rpc Get(GetDestination) returns (stream Update) {}
```

Comment:
I'm done fighting vscode on this stuff ;)

There's already a issue to check formatting in CI https://github.com/linkerd/linkerd2/issues/4936